perfmetrics/scripts/job_files/seq_rand_read_write_presubmit.fio

```diff
@@ -0,0 +1,119 @@
+[global]
+ioengine=libaio
+direct=1
+fadvise_hint=0
+verify=0
+rw=read
+bs=1M
+iodepth=64
+invalidate=1
+ramp_time=10s
+runtime=60s
+startdelay=5m
+time_based=1
+nrfiles=1
+thread=1
+openfiles=1
+group_reporting=1
+allrandrepeat=1
+filename_format=$jobname.$jobnum.$filenum
+
+[1_thread]
+stonewall
+startdelay=300
+bs=16k
+directory=gcs/256kb
+filesize=256k
+rw=read
+numjobs=40
+
+[2_thread]
+stonewall
+startdelay=670
+bs=16k
+directory=gcs/256kb
+filesize=256k
+rw=write
+numjobs=40
+
+[3_thread]
+stonewall
+startdelay=2520
+bs=16k
+directory=gcs/256kb
+filesize=256k
+rw=randread
+numjobs=40
+
+[4_thread]
+stonewall
+startdelay=2890
+bs=16k
+directory=gcs/256kb
+filesize=256k
+rw=randwrite
+numjobs=40
+
+[5_thread]
+stonewall
+startdelay=1040
+directory=gcs/50mb
+filesize=50M
+rw=read
+numjobs=40
+
+[6_thread]
+stonewall
+startdelay=1410
+directory=gcs/50mb
+filesize=50M
+rw=write
+numjobs=40
+
+[7_thread]
+stonewall
+startdelay=3260
+directory=gcs/50mb
+filesize=50M
+rw=randread
+numjobs=40
+
+[8_thread]
+stonewall
+startdelay=3630
+directory=gcs/50mb
+filesize=50M
+rw=randwrite
+numjobs=40
+
+[9_thread]
+stonewall
+startdelay=1780
```

Comment:
Can we reduce (if it doesn't affect the tests) start delay times for jobs so that overall it takes less time for presubmit tests to run ?

Reply:
fixed